docs/language/accounts.mdx

```diff
@@ -167,6 +167,16 @@ struct AccountKey {
 struct PublicKey {
```

Comment:
Can `PublicKey` fields (data, signature algo, isValid..) be updated after the creation of the object?

Reply:
No, they are defined as constant fields (i.e. read-only).